indicator/login/indicator.class.php

```diff
@@ -45,11 +45,31 @@ protected function get_rawdata($startdate, $enddate) {
         $params['courseid'] = $this->courseid;
         $params['startdate'] = $startdate;
         $params['enddate'] = $enddate;
-        $sql = "SELECT id, userid, time
-                FROM {log}
-                WHERE course = :courseid AND time >= :startdate AND time <= :enddate
-                ORDER BY time ASC";
-        if ($logs = $DB->get_recordset_sql($sql, $params)) {
+		// attempt to query both v2.7 logstore_standard_log table as well as legacy log table
+		try {
```

Comment:
This would be better get the list of logstore readers and using that to fallback on just querying the old log table if legacy log is enabled but standard isn't.